include/mbedtls/build_info.h

```diff
@@ -53,6 +53,12 @@
 #define _CRT_SECURE_NO_DEPRECATE 1
 #endif
 
+/* Define `inline` on some non-C99-compliant compilers. */
```

Comment:
There are quite a few more files that also have a redundant definition:

chacha20.c
ssl_misc.h
aria.c
debug.c
mps_reader.c
ecp.c
ecp_curves.c
poly1305.c